src/nvim/main.c

```diff
@@ -217,6 +218,8 @@ int main(int argc, char **argv)
     argv[i] = buf;
   }
 #endif
+  unsetenv("VIM");
+  unsetenv("VIMRUNTIME");
```

Comment:
```
src/nvim/main.c:221:  Use os_unsetenv(...) instead of unsetenv(...).  [runtime/os_fn] [2]
src/nvim/main.c:222:  Use os_unsetenv(...) instead of unsetenv(...).  [runtime/os_fn] [2]
```